satpy/tests/etc/composites/fake_sensor.yaml

```diff
@@ -241,3 +241,8 @@ composites:
         prerequisites:
           - name: ds13
             modifiers: ['mod1', 'mod_opt_only']
+  comp_multi:
+    compositor: !!python/name:satpy.tests.utils.FakeCompositor
+    prerequisites:
+      - ds1
+      - ds4_b
```

Comment:
Did this composite require a whole new test reader? Couldn't one of the existing work?

Reply:
Oh or is it because in the fake2 reader the datasets have the same name as the fake1 readers?

Reply:
I'm not sure.  It needs new datasets including an `area` attribute on each dataset, which seems to require lons/lats to be added.  I was reluctant to add those to `fake2` or `fake3` in case of unintended effects on existing tests.  Maybe I could have done without a new fake reader, but this seemed to be the safest alternative to me to be sure to not affect existing tests.

Reply:
Yeah it isn't great that another one had to be added, but I think this is for the best. If anything it will leave the door open to add more datasets to the fake4 reader in the future.